.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
+- [ ] Docstrings for all functions
```

Comment:
All well and true, but it's our responsibility as maintainers to ensure docstring style is up to date and consistent.

It's a good document, but it's just _far and away_ more than any new contributor needs. If a docstring is subtly incorrect or needs a new feature, again, it's our responsibility to point them in the right direction in the course of the PR.

I do strongly feel we don't want to front load this. I was on the fence about including any links at all.